rpc/src/v1/helpers/poll_filter.rs

```diff
@@ -49,7 +49,11 @@ impl SyncPollFilter {
 #[derive(Clone)]
 pub enum PollFilter {
 	/// Number of last block which client was notified about.
-	Block(BlockNumber),
+	Block {
+		last_block_number: BlockNumber,
+		#[doc(hidden)]
```

Comment:
why hidden in docs?

Reply:
Thanks - I was deliberating whether to hide it or not; I decided it was an implementation detail, but open to removing that.